PlayerInfo/PlayerInfo.cpp

```diff
@@ -49,116 +49,100 @@ namespace Plugin {
     /* virtual */ const string PlayerInfo::Initialize(PluginHost::IShell* service)
     {
         ASSERT(service != nullptr);
-        ASSERT(_player == nullptr);
         ASSERT(_service == nullptr);
+        ASSERT(_connectionId == 0);
```

Comment:
Pls also update the patch version macro to 2 at beginning of PlayerInfo.cpp. This will then return the updated version in calls to Controller.status.


#define API_VERSION_NUMBER_PATCH 1